recipes/deepspeed/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "deepspeed" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0ee39e1b01fe4ecbd89e7c4358db6327cee6fe52a201157999e1c995985453a4
+
+build:
+  noarch: python
```

Comment:
It sounds very much like this package will not be noarch (which also only can be built on linux AFAIK).